docs/language/functions/regexp_replace.md

```diff
@@ -10,28 +10,50 @@ regexp_replace(s: string, re: string|regexp, new: string) -> string
 ### Description
 
 The _regexp_replace_ function substitutes all characters matching the
-[regular expression](../overview.md#regular-expressions) `re` in string `s` with
+[regular expression](../overview.md#811-regular-expressions) `re` in string `s` with
 the string `new`.
 
 Variables in `new` are replaced with corresponding matches drawn from `s`.
-A variable is a substring of the form $name or ${name}, where name is a non-empty
-sequence of letters, digits, and underscores. A purely numeric name like $1 refers
-to the submatch with the corresponding index; other names refer to capturing
-parentheses named with the `(?P<name>...)` syntax. A reference to an out of range or
+A variable is a substring of the form `$name` or `${name}`, where `name` is a non-empty
+sequence of letters, digits, and underscores. A purely numeric name like `$1` refers
+to the submatch with the corresponding index; other names refer to parenthesized
+capturing groups named with the `(?P<name>...)` syntax. A reference to an out of range or
 unmatched index or a name that is not present in the regular expression is replaced
-with an empty slice.
+with an empty string.
```

Comment:
While [slices](https://zed.brimdata.io/docs/language/overview#77-slices) are present in Zed, I'm pretty sure we want to say "string" here.